src/shared-components/tooltip/__snapshots__/test.js.snap

```diff
@@ -16,15 +16,16 @@ exports[`<Tooltip /> UI snapshot renders content and children 1`] = `
   cursor: pointer;
 }
 
-.emotion-2 {
+.emotion-6 {
+  max-width: none;
   top: 120%;
-  background: #ffffff;
-  border: 1px solid #ededf0;
-  box-shadow: 0 12px 20px 0 rgba(51,46,84,0.05);
-  color: #706D87;
+  background: #332e54;
+  box-shadow: 0px 8px 24px rgba(51,46,84,0.05);
+  border-radius: 0.5rem;
```

Comment:
@daigof can you make the border radius `4px` when tooltip `isSmall`? Then it lgtm!